day1/04_Commits/04_History.md

```diff
@@ -55,7 +55,7 @@ also compare specific commits.
  * Use `git log` to print the current history
  * Use `git show` to show specific commits (defaults to the latest)
  * Use `git diff` to compare changes between specific revisions
- * Use `git blame .gitignore` to see the authors for the file
+ * Use `git blame .gitignore to see the files` authors
```

Comment:
`git blame .gitignore` is the command we want to highlight, so the backtick needs to stay there :)

Reply:
updated

Reply:
We're almost there: All we need is an ' in `file's` so the grammar works out and we're there!

Context, courtesy of ChatGPT, because I also did it wrong in the original issue:
"files authors" should be corrected to "file's authors" because you are referring to the authors of a specific file, not multiple files. The possessive form "file's" indicates that you are talking about the authors of a single file.

Reply:
ok

Reply:
updated pls check @theFeu